### PR TITLE
XVI-2 clauses

### PR DESCRIPTION
**Pull request by ColinMaudry**
I used the "Follow the guidance" when only part of the targeted guidance was related to the present article/clause.

I wonder if we could phrase theses references more accurately.

### CODE REVIEW

File: docs/gpa.md [hunk 1]
```diff
@@ -475,47 +475,56 @@ This requires the [Participation Fees](https://github.com/open-contracting-exten
         <td>a description of the goods or services procured;</td>
         <td markdown=1>
 
-* 
+* Follow the guidance for [XIII:2](#XIII:2).
```
Comment (jpmckinney) [inline docs/gpa.md:478]: I agree, like in your issue description, that it's hard to follow the reference. For this article, let's instead repeat the specific guidance, and mention that award notices are also discussed in XIII:2.

Comment (ColinMaudry) [reply docs/gpa.md:478]: OK.
I forgot to write guidance for "lowest offers taken into account in   the award of the contract".

Comment (ColinMaudry) [reply docs/gpa.md]: Done both

**Comment by jpmckinney** [comment]
Looks great!
